light-client/src/fork_detector.rs

```diff
@@ -52,10 +52,8 @@ pub trait ForkDetector: Send {
 /// the given trusted state, and then:
 ///
 /// - If the verification succeeds, we have a real fork
-/// - If verification fails because of lack of trust,
-///   we have a potential fork.
-/// - If verification fails for any other reason, the
-///   witness is deemed faulty.
+/// - If verification fails because of lack of trust, we have a potential fork.
+/// - If verification fails for any other reason, the witness is deemed faulty.
```

Comment:
We actually never enforce the rules we have via CI by running fmt via nightly at least there.
See: https://github.com/informalsystems/tendermint-rs/blob/a556d77e2a4b92e6d7d653c8784a281cceca3388/.rustfmt.toml#L1-L12

Reply:
we should probably change this.

Reply:
Yeah let's remove the nightly-only formatting directives.

Reply:
ref: https://github.com/informalsystems/tendermint-rs/issues/153 and https://github.com/informalsystems/tendermint-rs/pull/121

Either remove them or reject PRs via CI that do not comply with these rules.